Sources/MapboxNavigation/UserCourseView.swift

```diff
@@ -79,11 +70,8 @@ open class UserPuckCourseView: UIView, CourseUpdatable {
     }
     
     // Sets the color on the user puck in 'stale' state. Puck will gradually transition the color as long as location updates are missing
-    @objc public dynamic var stalePuckColor: UIColor = #colorLiteral(red: 0.6000000238, green: 0.6000000238, blue: 0.6000000238, alpha: 1) {
-        didSet {
-            puckView.stalePuckColor = stalePuckColor
-        }
-    }
+    @available(*, deprecated, message: "No stale status in active navigation anymore.")
+    @objc public dynamic var stalePuckColor: UIColor = #colorLiteral(red: 0.6000000238, green: 0.6000000238, blue: 0.6000000238, alpha: 1)
```

Comment:
Please update `DayStyle` to stop using this property.

Reply:
Thanks for the suggestion, I'll update the style part.